settings.py

```diff
@@ -18,3 +18,5 @@
 USE_TZ = False
 
 INSTALLED_APPS = ("db",)
+
+ALLOWED_HOSTS = ["*"]
```

Comment:
Setting `ALLOWED_HOSTS` to `['*']` is not recommended for production as it allows any host to access the application. Specify the allowed hosts explicitly to enhance security.